.github/workflows/ci-build.yml

```
@@ -0,0 +1,25 @@
+name: CI Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      fail-fast: true
+    runs-on: ubuntu-22.04
```

```suggestion
    runs-on: ubuntu-latest
```
22.04 is the default.